platform/contracts/admin/src/msg.rs

```diff
@@ -6,7 +6,7 @@ use sdk::{
     schemars::{self, JsonSchema},
 };
 
-use crate::contracts::{
+pub use crate::contracts::{
```

Comment:
not sure I understand the motivation behind this, could you elaborate?

Reply:
It was not possible to access the type from outside, thus any contract and/or test that had to interact with it was blocked from doing so.